app/src/main/res/layout/fragment_record.xml

```diff
@@ -16,19 +16,20 @@
         android:layout_centerHorizontal="true"
         android:src="@drawable/ic_mic_white_36dp" />
 
-    <Chronometer
+    <TextView
```

Comment:
This change is not related to the PR (it's not translation). Please revert it or create a new PR.

Reply:
oh.....my mistake.
So I reset my repository.
I thought sending PR only my previous commit( Create strings.xml for South Korea ). but continues to add commit in PR....
I'll be careful

thank you feedback. I try it!